client/permission.go

```diff
@@ -0,0 +1,26 @@
+package client
+
+import (
+	"fmt"
+
+	clientv1 "github.com/spectrocloud/palette-sdk-go/api/client/v1"
+	"github.com/spectrocloud/palette-sdk-go/api/models"
+)
+
+// GetPermissionByName retrieves an existing permission by name and permissionScope(project, tenant & resource).
+func (h *V1Client) GetPermissionByName(permissionName string, permissionScope string) (*models.V1Permission, error) {
```

Comment:
Please make permissionScope a new string type with consts for the 3 possible values.

Reply:
done